Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM public.ecr.aws/amazonlinux/amazonlinux
+LABEL maintainer="amazon"
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# create directory for the app user
+RUN mkdir -p /home/app
+ENV HOME=/home/app
+ENV APP_HOME=$HOME/workdir
+ENV PYTHONPATH=$APP_HOME/plugins/
+RUN mkdir $APP_HOME
+WORKDIR $APP_HOME
+
+RUN yum update -y
+RUN yum install -y python37 gcc gcc-g++ python3-devel
+RUN yum install -y java-1.8.0-openjdk unixODBC-devel
+RUN yum install -y zip unzip bzip2 gzip
+RUN yum install -y gcc-c++ cyrus-sasl-devel libcurl-devel openssl-devel shadow-utils
+RUN yum clean all
+
+RUN python3 -m pip install --upgrade pip
+
+# Install wheel to avoid legacy setup.py install
+RUN pip3 install wheel
+
+# On RHL and Centos based linux, openssl needs to be set as Python Curl SSL library
+ENV PYCURL_SSL_LIBRARY=openssl
+RUN pip3 install --upgrade pip
```

Comment:
What is the difference with the previous command?:
```
RUN python3 -m pip install --upgrade pip
```

Reply:
I'm not sure this docker file is inspired from MWAA local runner docker project. May be it is related to what was said about openssl in line 27.

Reply:
Removed